perf/_system.py

```diff
@@ -143,7 +143,8 @@ def read_msr(self, cpu, reg_num, use_warnings=False):
         try:
             fd = os.open(path, os.O_RDONLY)
             try:
-                data = os.pread(fd, size, reg_num)
+                os.lseek(fd, reg_num, os.SEEK_SET)
```

Comment:
I would prefer to use pread() if available:

if hasattr(os, 'pread'):
   ... pread ...
else:
  ... seek ...
  ... read ...